drivers/util.py

```diff
@@ -977,6 +977,22 @@ def dom0_disks():
     SMlog("Dom0 disks: %s" % disks)
     return disks
 
+def set_scheduler_sysfs_node(node, str):
+    """Set the scheduler for a sysfs node (e.g. '/sys/block/sda')"""
```

Comment:
This comment doesn't match the usage above `/sys/dev/block/x:x`, maybe have the two expected forms?

Reply:
sysfs has many different symlinks pointing to the same sysfs node (`/sys/block/sda` itself is actually a symlink). I'm don't think that it is useful or necessary to list an extra form.